Packs/CommonScripts/Scripts/SetGridField/SetGridField.py

```diff
@@ -362,24 +369,27 @@ def main():  # pragma: no cover
                                    keys_from_nested=argToList(args.get('keys_from_nested'))
                                    )
         # Execute automation 'setIncident` which change the Context data in the incident
-        res = demisto.executeCommand("setIncident", {
+        res_set = demisto.executeCommand("setIncident", {
             'customFields': {
                 grid_id: table,
             },
         })
         # we want to check if the incident was succefully updated
         # we execute command and not using `demisto.incident()` because we want to get the updated incident and context
         res = demisto.executeCommand("getIncidents", {"id": demisto.incident().get("id")})
-        data = res[0]["Contents"]["data"]
-        custom_fields = data[0].get("CustomFields")
-
-        if table and grid_id not in custom_fields:
-            raise ValueError(f"The following grid id was not found: {grid_id}. Please make sure you entered the correct "
+        is_playground = demisto.incident().get("isPlayground")
+        custom_fields = {}
+        for entry in res:
```

Comment:
Why do we iterating over the incidents? we will only get one, the one with the correct id given.

Reply:
We do not iterate over the the incident we do iterate over the response, it is an implementation base on Shahaf bug, 
He introduced it in the daily.